scout/server/blueprints/variants/templates/variants/indicators.html

```diff
@@ -61,20 +61,22 @@
   {% endif %}
 {% endmacro %}
 
-{% macro evaluations_badge(variant) %}
-  {% if variant.evaluations %}
-    {# Create a dictionary to count classifications #}
+{% macro generic_evaluation_badge(evaluations) %}
+  {% if evaluations %}
     {% set classification_counts = {} %}
-    {% for evaluation in variant.evaluations %}
-      {% set short = evaluation.classification.short %}
-      {% set label = evaluation.classification.label %}
-      {% set classification_counts = classification_counts.update({short: {
-        'count': (classification_counts[short].count if classification_counts[short] is defined else 0) + 1,
-        'label': label
-      }}) or classification_counts %}
+    {% for evaluation in evaluations %}
+      {% set c = evaluation.classification if evaluation.classification is defined else evaluation.ccv_classification %}
```

Comment:
There is a slight risk this will need to be split again if we start having any complicated cases with both kinds. But so far I think not, so all good.

Reply:
Mmm I think we need a fix in that sense. We should collect both ACMG and CCV for cancer variants. I'll fix

Reply:
Agreed!